installer/templates/sandcastle-admin.sh

```
@@ -0,0 +1,533 @@
+  set -a
+  # shellcheck source=/dev/null
+  source "$main_env"
+  set +a
+
+  # ── Step 3 & 4: Database restore ──────────────────────────────────────────
+
+  if [ "$skip_db" = false ]; then
+    info "Step 3/7: Starting PostgreSQL..."
+    $DOCKER compose -f "$COMPOSE_FILE" up -d postgres
+    local i
+    for i in $(seq 1 30); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)